.github/workflows/ci.yml

```diff
@@ -186,7 +186,7 @@ jobs:
           - suite-1
           - suite-2
           - suite-3
-          # suite-4 does not exist
+          - suite-4
```

Comment:
I don't think we should run this suite for all envronments (hdp3, and cdh5). It is time expensive.

Reply:
the suite-4 took 7 minute for each of the configurations.... Still there isn't much added value for running it 3 times.

... but it failed.
could it be the queries were not updated when we changed the char/varchar coercion rules?

Reply:
q85 seems to failing since fbc313e1bbdb0dbf3d0be04b445cd7a5b4fdc176 .
q72 seems to be failing since much before
test results fixed in #4419